backend/tests/utils/test_file_validator.py

```
@@ -0,0 +1,230 @@
+"""
+Tests for file validation utilities.
+"""
+
+import pytest
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)